src/collisions/COLLISIONS_compose_nl.F

```diff
@@ -58,9 +58,13 @@ subroutine COLLISIONS_compose_nl(G_in)
       do ibp=COLL_bands(1),COLL_bands(2)
        do ib=COLL_bands(1),COLL_bands(2)
          !
-         if (HXC_COLL_element(i_coll_mem)%table(ib,ibp,i_kmq)=="t") then
-           RT_Vnl_xc(i_n,i_m,i_k_NL,i_spin)=RT_Vnl_xc(i_n,i_m,i_k_NL,i_spin) &
-&                                               -cI*HXC_COLL_element(i_coll_mem)%v_c(i_comp)*G_in(ib,ibp,i_kmq)
+         if (HXC_COLL_element(i_coll_mem)%table(ib,ibp,i_kmq)=='t') then
+           if(COLLISIONS_load_SP) then
+             COLL=dcmplx(HXC_COLL_element(i_coll_mem)%v_c_SP(i_comp))
```

Comment:
COLL is of kind SP, why is it defined 
`COLL=dcmplx(X)`  ?
If SP is correct in the definition, it should be
`COLL=cmplx(X,kind=SP)`

Reply:
Yes, fine, I can change it